test/channel_permissions.js

```diff
@@ -861,6 +1019,42 @@ describe('Gaming', function() {
 		);
 	});
 
+	describe('Create message on frozen channel', function() {
+		before(async () => {
+			const channel = ctx.authClient.channel(ctx.channelType, ctx.channelId, {});
+			await channel.update({ frozen: true });
+			await removeUseFrozenChannelPermissions(ctx);
+		});
+
+		after(async () => {
+			const channel = ctx.authClient.channel(ctx.channelType, ctx.channelId, {});
+			await channel.update({ frozen: false });
+			await removeUseFrozenChannelPermissions(ctx);
+		});
+
+		it("admins can't send messages to a frozen channel", async () => {
+			await createMessageFrozen(ctx, ctx.adminUser, true);
+		});
+
+		it("admins can't create / delete reply to messages in a frozen channel", async () => {
+			await createDeleteReactionFrozen(ctx, ctx.adminUser, true);
+		});
+
+		describe('after admins get the UseFrozenChannelPermission', () => {
+			before(async () => {
+				await addUseFrozenChannelPermissions(ctx);
+			});
+
+			it('admins can create a message', async () => {
```

Comment:
how about delete ?

Reply:
Deleting is possible on frozen channels without the new permission